core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteractionEditorDirective.js

```diff
@@ -463,19 +500,20 @@ oppia.directive('testInteractionPanel', [
       scope: {
         stateContent: '&',
         inputTemplate: '&',
-        onSubmitAnswer: '&'
+        onSubmitAnswer: '&',
```

Comment:
Just to check -- is this directive actually used?

Reply:
Yeah, I searched and it was used in training data editor directive.